courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb

```diff
@@ -34,36 +34,10 @@
    "metadata": {},
```

Comment:
<p>Line 1, replace with with:  &nbsp;Let's convert the Yes to 1 and No to 0. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
Updated the same.